koap/js_tests/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "js_tests",
+  "version": "1.0.0",
+  "description": "Tests for the js target",
+  "main": "index.js",
+  "scripts": {
+    "build-and-test": "sh prepare-testing.sh",
+    "test": "jest",
+    "build-dev": "webpack --mode=development"
+  },
+  "jest": {
+    "preset": "ts-jest",
+    "setupFiles": [
+      "./helpers.js"
+    ]
+  },
+  "author": "Justin Tiffner",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.12.3",
+    "@babel/preset-env": "^7.12.1",
+    "@babel/preset-typescript": "^7.12.1",
+    "@types/jest": "^26.0.15",
+    "babel-jest": "^26.6.2",
+    "babel-loader": "^8.2.1",
+    "jest": "25.5.4",
```

Comment:
I was getting warnings from ts-jest that it hadn't been tested with jest 26+ and to move back to <26.

Same thing with typescript 4.